lib/protocol/networks.js

```diff
@@ -770,18 +770,18 @@ regtest.txStart = 0;
 
 regtest.names = {
   auctionStart: 0,
-  rolloutInterval: 1000,
+  rolloutInterval: 2,
```

Comment:
Can you leave these to the previous values? We've modified these numbers compared to upstream